modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/KotlinClientCodegen.java

```diff
@@ -963,6 +963,38 @@ public ModelsMap postProcessModels(ModelsMap objs) {
         return objects;
     }
 
+    @Override
+    public Map<String, ModelsMap> postProcessAllModels(Map<String, ModelsMap> objs) {
+        objs = super.postProcessAllModels(objs);
+        if (getSerializationLibrary() == SERIALIZATION_LIBRARY_TYPE.kotlinx_serialization) {
+            for (Map.Entry<String, ModelsMap> modelsMap : objs.entrySet()) {
+                for (ModelMap mo : modelsMap.getValue().getModels()) {
```

Comment:
minor suggestion: add some comments in this for loop to explain what it tries to archive